### PR TITLE
update guzzlehttp/guzzle to 6.3.0

### PR DESCRIPTION
**Pull request by icewind1991**
requires:

- [x] https://github.com/nextcloud/3rdparty/pull/83

Needed for updating the openstack sdk

Fixes https://github.com/nextcloud/3rdparty/issues/49

### CODE REVIEW

**Comment by MorrisJobke** [comment]
CI fails

**Comment by icewind1991** [comment]
ci looks happy, review time

**Review by rullzer — APPROVED** [review]
Seems to all still work as far as I can tell.

File: build/integration/features/bootstrap/BasicStructure.php [hunk 1]
```diff
@@ -249,13 +253,13 @@ public function theHTTPStatusCodeShouldBe($statusCode) {
...
 	public function theContentTypeShouldbe($contentType) {
-		PHPUnit_Framework_Assert::assertEquals($contentType, $this->response->getHeader('Content-Type'));
+		PHPUnit_Framework_Assert::assertEquals($contentType, $this->response->getHeader('Content-Type')[0]);
 	}
 
 	/**
 	 * @param ResponseInterface $response
 	 */
-	private function extracRequestTokenFromResponse(ResponseInterface $response) {
+	private function extracRequestTokenFromResponse(\Psr\Http\Message\ResponseInterface $response) {
```
Comment (nickvergessen) [inline build/integration/features/bootstrap/BasicStructure.php]: Either this is not needed, or the docs is now wrong

Comment (icewind1991) [reply build/integration/features/bootstrap/BasicStructure.php]: Fixed

**Comment by oparoz** [comment]
All ready to merge?

**Review by MorrisJobke — APPROVED** [review]

**Pull request merged**